k8s/oidc-vault/k8s/server-statefulset.yaml

```
@@ -71,12 +71,21 @@ spec:
             readOnly: false
           readinessProbe:
             httpGet:
-              path: /keys # TODO: Change this to /ready when using 1.5.2+
+              path: /ready
               port: 8008
             failureThreshold: 5
             initialDelaySeconds: 5
             periodSeconds: 5
             timeoutSeconds: 3
+          livenessProbe:
+            httpGet:
+              path: /ready
```

There are separate liveness and readiness endpoints:
```suggestion
              path: /live
```